index.html

```diff
@@ -242,53 +277,57 @@ <h2 class="about-me-text">About me</h2>
-        <li class="about-head-text-2">Ruby on Rails</li>
-        <li class="about-head-text-3">RSPec</li>
-        <li class="about-head-text-4">CapyBara</li>
-        <li class="about-head-text-5">Selenium</li>
+      <ul class="about-me-skills2">
+        <li class="about-head-text-1 skill-1">Bootstrap</li>
+        <li class="about-head-text-2 skill-2">Ruby on Rails</li>
+        <li class="about-head-text-3 skill-3">RSPec</li>
+        <li class="about-head-text-4 skill-4">CapyBara</li>
+        <li class="about-head-text-5 skill-5">Selenium</li>
 
         <br> <br>
       </ul>
         <!-- skills list -->
-        <li class="about-heading-skills"> <img src="./images/circle.png" alt="circle logo" class="circle-img"> 
+        <li class="about-heading-skills abt-head3"> <img src="./images/circle.png" alt="circle logo" class="circle-img"> 
         Skills
         </li>
-       <ul>
+       <ul class="about-me-skills3">
           <li class="about-head-text-1">Codekit</li>
-          <li class="about-head-text-2">Github </li>
-          <li class="about-head-text-3">Codepen</li>
-          <li class="about-head-text-4">Gitlab </li>
-          <li class="about-head-text-5">Terminal </li>
+          <li class="about-head-text-2 skill3-1">Github </li>
+          <li class="about-head-text-3 skill3-2">Codepen</li>
+          <li class="about-head-text-4 skill3-1">Gitlab </li>
+          <li class="about-head-text-5 skill3-5">Terminal </li>
         </ul>
       </ul>
-
-      <!-- contact me form -->
- 
-      <footer>
+      <footer class="footer-bottom">
         <div class="contact-me">
           <h2 class="contact-me-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
         
         <form method="POST" action="https://formspree.io/f/xzbqpdra">
           <ul class="form-items-list">
 
+
             <li class="no-outline">
               <input class="Name" type="text" name="name" placeholder="Full Name" maxlength="30" required>
             </li>   
+            <li class="last-name">
+              <input type="text" class="lname" name="last name" placeholder="Last Name" maxlength="30">
+            </li>
 
             <li class="no-outline">
               <input class="email" type="email" name="email" placeholder="Email Address" required>
             </li>
 
             <li>
               <textarea class="text-area" name="text-area" placeholder="Enter Text Here" cols="30" rows="4" maxlength="500" required></textarea>
-            </li>   
-              
+            </li>
+
+            <li> <textarea  class="text-area-desktop" name="text-area" placeholder="Hello Mariane. I would like to get in touch with you..." cols="30" rows="4" maxlength="500"></textarea>
+            </li>  
+                
          <!-- get in touch button -->
           <li>
            <button type="submit" class="button4">Get In Touch</button></li>
```

Comment:
- [ ] Let's also make sure to make the contact me section centered on the page, currently it is more towards the left please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230709219-3eb25467-9a5a-4b73-aa4e-2cbdfa648acd.png)

Figma Desing :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230709372-6c07288f-b135-4c97-8e09-d0b1c827802b.png)

- [ ] [OPTIONAL] As mentioned in the project requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230709295-d277c017-7634-4eab-b9e2-acc630394144.png)
You can use only one input field for the full name, rather than two different first and last name inputs, this will make it easier for you and also will be a good user experience